src/components/home/transaction-content.tsx

```diff
@@ -257,7 +258,7 @@ const TransactionsContent = ({
               style={{
```

Comment:
Please do not use inline style

Reply:
Inline style are everywhere we need to handle in each file. We cannot hold release because of inline styling